### ISSUE TITLE
Fix `clone.test.ts` – can't find file `test-walk/folder/3.txt/`

### ISSUE THREAD

**Issue by felixhaeberle** [inlang/inlang#753]
## Problem

A clone test seems to be broken and can't find a file.

```sh
@inlang-git/client:test:  ✓ src/raw/tests/GitRefManager.test.ts  (4 tests) 33ms
@inlang-git/client:test: stdout | src/raw/tests/clone.test.ts > clone > should set up the remote tracking branch by default
@inlang-git/client:test: FilesystemError: ENOENT: No such file or directory, symlink '/src/raw/tests/fixtures/isomorphic-git/__tests__/__fixtures__/test-walk/folder/3.txt/'
@inlang-git/client:test:     at Object.symlink (file:///home/runner/work/inlang/inlang/source-code-git/fs/dist/implementations/memoryFs.js:132:23)
@inlang-git/client:test:     at FileSystem.writelink (file:///home/runner/work/inlang/inlang/source-code-git/client/node_modules/isomorphic-git/internal-apis.js:4841:17)
@inlang-git/client:test:     at /home/runner/work/inlang/inlang/source-code-git/client/node_modules/isomorphic-git/index.cjs:6146:28
@inlang-git/client:test:     at processTicksAndRejections (node:internal/process/task_queues:95:5)
@inlang-git/client:test:     at async Promise.all (index 1404)
@inlang-git/client:test:     at /home/runner/work/inlang/inlang/source-code-git/client/node_modules/isomorphic-git/index.cjs:6118:7
@inlang-git/client:test:     at /home/runner/work/inlang/inlang/source-code-git/client/node_modules/isomorphic-git/index.cjs:987:16 {
@inlang-git/client:test:   code: 'ENOENT',
@inlang-git/client:test:   path: '/src/raw/tests/fixtures/isomorphic-git/__tests__/__fixtures__/test-walk/folder/3.txt/',
@inlang-git/client:test:   syscall: 'symlink',
@inlang-git/client:test:   target: undefined
@inlang-git/client:test: }
@inlang-git/client:test: 
```

### Expected behavior

Test should find the file.

Maybe it's because the 3.txt with a slash behind it is registered as a folder and not file? 
Only a wild guess.

## Steps to reproduce

run `npm run test`.

**Comment by samuelstroschein** [comment]
Interesting. The tests pass even though this error is thrown.

**Comment by felixhaeberle** [comment]
@samuelstroschein Yep this is a configuration isn't it?

**Comment by araknast** [comment]
Got it, should be fixed by merging #759
